src/mococrw/hsm.h

```diff
@@ -86,8 +86,8 @@ class HsmEngine : public HSM
 public:
     /**
      * @brief Constructor for an object that can manage keys on HSM using OpenSSL Engine
-     * @note Each HsmEngine object is associated with a specific token and a pin to login to that
-     * token
+     * @note Each HsmEngine object is associated with a specific token
+     * @warning PIN value is not safely cleaned from memory. Make sure you clean it.
```

Comment:
How do they clean it - please help the user? Why did we remove the cleanse wrapped function? Couldn't the users use that for convenience?

Reply:
Done